src/wallet/coinselection.cpp

```diff
@@ -244,7 +244,10 @@ bool KnapsackSolver(const CAmount& nTargetValue, std::vector<OutputGroup>& group
         return true;
     }
 
-    if (nTotalLower < nTargetValue) {
+    // we prioritize the lowest larger if
```

Comment:
This feels like a completely unrelated change to the coin selection algorithm.

Reply:
Dropped the relevant commit. I will make a PR once I have simulation results to back the change after merge.